src/common/net/fetch-client.ts

```diff
@@ -151,13 +200,74 @@ export class FetchHttpClient extends HttpClient implements HttpClientInterface {
 
     return new FetchHttpClientResponse(res);
   }
+
+  private async fetchRequestWithRetry(
+    url: string,
+    method: string,
+    body?: any,
+    headers?: RequestHeaders,
+  ): Promise<HttpClientResponseInterface> {
+    let response: HttpClientResponseInterface;
+    let requestError: any = null;
```

Comment:
We should declare `requestError` inside of `makeRequest`. It doesn't need to be outside the scope of that method.